tensorflow-core/tensorflow-core-api/src/bazel/op_generator/op_specs.cc

```diff
@@ -187,6 +190,15 @@ string SnakeToCamelCase(const string& str, bool upper = false) {
       result += c;
     }
   }
+
+  if(result == "for"){
+    return "for_";
+  } else if(result == "if"){
+    return "if_";
+  } else if(result == "while"){
+    return "while_";
+  }
+
```

Comment:
I'm planning to merge the [Java op generator](https://github.com/tensorflow/java/pull/244) before this one so can you please make sure it contains these changes so we can revert them here?

Reply:
Yeah, it will do that (although I went with `*Output` instead of `*_`).